### PR TITLE
Update python installation instructions

### PR DESCRIPTION
**Pull request by Dr-Electron**
# Description of change

Updated the docs to use the PyPI package

## Type of change

- Documentation Fix

## Change checklist

- [x] I have followed the contribution guidelines for this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that new and existing unit tests pass locally with my changes

### CODE REVIEW

File: documentation/docs/getting_started/python.mdx [hunk 1]
```diff
@@ -18,18 +18,31 @@ import WarningPasswordStorage from '../_admonitions/_password_storage.md';
 
 * [Python 3.x](https://www.python.org).
 * [pip](https://pypi.org/project/pip).
-* [Rust and Cargo](https://doc.rust-lang.org/cargo/getting-started/installation.html) to compile the binding.
-* (for Linux only) `libudev`. You can install it with `apt install libudev-dev`.
 
+## Install package from PyPI
+
+You can install the library by executing `pip install iota-client`
```
Comment (thibault-martinez) [inline documentation/docs/getting_started/python.mdx]:
```suggestion
You can install the library by executing `pip install iota-client`.
```

File: documentation/docs/getting_started/python.mdx [hunk 2]
```diff
@@ -43,7 +56,7 @@ cd iota.rs/client/bindings/python
 
 ### Install the Required Dependencies and Build the Wheel
 
-Once you have [cloned the library](#clone-the-library),
+Once you have [cloned the library](#clone-the-repository),
```
Comment (thibault-martinez) [inline documentation/docs/getting_started/python.mdx]: Change the text as well ?

Comment (Dr-Electron) [reply documentation/docs/getting_started/python.mdx]: Yes, totally forgot. Thanks

Comment (Dr-Electron) [reply documentation/docs/getting_started/python.mdx]:
```suggestion
Once you have [cloned the repository](#clone-the-repository),
```

**Review by thibault-martinez — DISMISSED** [review]

**Review by Thoralf-M — APPROVED** [review]

**Review by thibault-martinez — APPROVED** [review]

**Pull request merged**